overlord/snapstate/snapstate.go

```diff
@@ -153,6 +167,17 @@ func doInstall(s *state.State, snapst *SnapState, ss *SnapSetup) (*state.TaskSet
 			prev = tasks[len(tasks)-1]
 		}
 
+		// discard the first As in A-...-A refreshes
```

Comment:
This comment is a bit hard to read. I understand it now, but I probably won't understand it in 6 months. Would be nice to be more descriptive of the issue being solved.